src/data/behandlendeenhet/behandlendeEnhet.ts

```diff
@@ -1,46 +1,46 @@
 import { Reducer } from "redux";
 import {
-  HENT_BEHANDLENDE_ENHET_FEILET,
-  HENTER_BEHANDLENDE_ENHET,
-  BEHANDLENDE_ENHET_HENTET,
+  BehandlendeEnhetActions,
+  BehandlendeEnhetActionTypes,
 } from "./behandlendeEnhet_actions";
 import { BehandlendeEnhet } from "./types/BehandlendeEnhet";
+import { ApiError } from "../../api/axios";
 
 export interface BehandlendeEnhetState {
   henter: boolean;
-  hentingFeilet: boolean;
+  error: ApiError | null;
```

Comment:
kan evt endre til `error?: ApiError` så slipper man sette error til null i initialState, hvis ikke det er gode grunner for å bruke `null` framfor `undefined`. Gjelder alle redux-states med error.

Reply:
Fikset